libexec/powprox-print-pow-config

```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env bash
-pow_bin="$HOME/Library/Application Support/Pow/Current/bin"
-exec "$pow_bin/node" "$pow_bin/pow" --print-config
+if [ -f /usr/local/bin/pow ]; then
+  pow --print-config
+else
+  pow_bin="$HOME/Library/Application Support/Pow/Current/bin"
+  exec "$pow_bin/node" "$pow_bin/pow" --print-config
+fi
```

Comment:
How about flipping this conditional? Check for a mainline Pow install first, then fall back to just relying on `$PATH`:
```bash
pow_bin="$HOME/Library/Application Support/Pow/Current/bin"
if [ -d "$pow_bin" ]; then
  exec "$pow_bin/node" "$pow_bin/pow" --print-config
else
  exec pow --print-config
fi
```